pipelines/roles/maven-repo-role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright contributors to the Galasa project 
```

Comment:
Has this Copyright message been approved by legal ?
Normally I thought they had dates in..etc.

Reply:
@hobbit1983 Can you provide some insight on this please? I assumed the copyright block had been approved as we have been using it for some time now

Reply:
@jadecarino @techcobweb yes this has changed quite a few times. The last review of the copyright headers by legal resulted in the one we are now using